File: services/identity-service/src/main/java/br/com/deltaglobalbank/identity/features/ipAllowlist/tenant/createTenantIpAllowlist/CreateTenantIpAllowlistRequest.java
package br.com.deltaglobalbank.identity.features.ipAllowlist.tenant.createTenantIpAllowlist;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record CreateTenantIpAllowlistRequest(
    @NotBlank String cidr,
    @Size(max = 255) String description
) {
}

File: services/identity-service/src/main/kotlin/br/com/deltaglobalbank/identity/features/ipAllowlist/tenant/createTenantIpAllowlist/CreateTenantIpAllowlistRequest.kt
package br.com.deltaglobalbank.identity.features.ipAllowlist.tenant.createTenantIpAllowlist

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size


data class CreateTenantIpAllowlistRequest (
    @field:NotBlank
    var cidr: String,
    @field:Size(max = 255)
    var description: String? = null
)